aiopg/pool.py

```diff
@@ -255,10 +258,13 @@ def release(self, conn):
                     tran_status)
                 conn.close()
                 return fut
+
             if self._closing:
                 conn.close()
-            else:
+            elif conn.closed:
```

Comment:
Can you elaborate here? bit not follow in what case we need this?

Reply:
i just realized the fundamental premise of this function is to reuse connections, so there's a problem when we're passing connections that are in a bad state (thus I reverted last change).

Here's the issue which we need to resolve: conn.close() consistently throws an exception, how do we drop it?  Right now we're going to call release, which will add it back to the free pool.  If we don't call release it will be stuck in the used pool.

Reply:
Hold on but I thought that `self._cur.close()` throws exception in your case not `conn.close()` ?